server_statvfs_darwin.go

```diff
@@ -6,16 +6,19 @@ import (
 
 func statvfsFromStatfst(stat *syscall.Statfs_t) (*StatVFS, error) {
 	return &StatVFS{
-		Bsize:   uint64(stat.Bsize),
-		Frsize:  uint64(stat.Bsize), // fragment size is a linux thing; use block size here
-		Blocks:  stat.Blocks,
-		Bfree:   stat.Bfree,
-		Bavail:  stat.Bavail,
-		Files:   stat.Files,
-		Ffree:   stat.Ffree,
-		Favail:  stat.Ffree,                                                      // not sure how to calculate Favail
-		Fsid:    uint64(uint64(stat.Fsid.Val[1])<<32 | uint64(stat.Fsid.Val[0])), // endianness?
-		Flag:    uint64(stat.Flags),                                              // assuming POSIX?
-		Namemax: 1024,                                                            // man 2 statfs shows: #define MAXPATHLEN      1024
+		Bsize:  uint64(stat.Bsize),
+		Frsize: uint64(stat.Bsize), // fragment size is a linux thing; use block size here
+		Blocks: stat.Blocks,
+		Bfree:  stat.Bfree,
+		Bavail: stat.Bavail,
+		Files:  stat.Files,
+		Ffree:  stat.Ffree,
+		Favail: stat.Ffree,                                              // not sure how to calculate
+		Fsid:   uint64(stat.Fsid.Val[1])<<32 | uint64(stat.Fsid.Val[0]), // endianness?
+		Flag:   uint64(stat.Flags),                                      // assuming POSIX?
+
+		// statvfs.c: statvfs.namemax = NAME_MAX
+		// syslimits.h: #define NAME_MAX 255
+		Namemax: 255,
```

Comment:
Looked all this up in their open source on how they’re filling in these values. Only big thing is that `MAXPATHLEN` is the maximum total path, while `NAME_MAX` is the longest individual directory entry name allowed.